upgradeable_contract/new_contract/src/lib.rs

```diff
@@ -1,20 +1,31 @@
 #![no_std]
 
-use soroban_sdk::{contract, contractimpl, contracttype, Address, BytesN, Env};
+use soroban_sdk::{contract, contracterror, contractimpl, contracttype, Address, BytesN, Env};
 
 #[contracttype]
 #[derive(Clone)]
 enum DataKey {
     Admin,
 }
 
+#[contracterror]
+#[derive(Copy, Clone, Debug, Eq, PartialEq, PartialOrd, Ord)]
+#[repr(u32)]
+pub enum Error {
+    AlreadyInitialized = 1,
```

Comment:
nit: Please add semicolon

Reply:
You mean, semicolon after the enum closing bracket?